lib/generators/htmx/install_generator.rb

```diff
@@ -40,38 +46,34 @@ def manifest(javascript_dir)
         Pathname.new(destination_root).join(javascript_dir, 'application.js')
       end
 
-      def setup_importmap
-        run 'bin/importmap pin htmx.org'
-
-        manifest = manifest('app/javascript')
-
+      def add_to_manifest(manifest, text)
         if manifest.exist?
-          append_file manifest, "\n#{IMPORTMAP_SETUP}"
+          append_file manifest, "\n#{text}"
         else
-          create_file manifest, IMPORTMAP_SETUP
+          create_file manifest, text
         end
       end
 
-      def setup_sprockets
-        manifest = manifest('app/assets/javascripts')
+      def setup_bun
+        run 'bun add htmx.org'
 
-        if manifest.exist?
-          append_file manifest, "\n#{SPROCKETS_SETUP}"
-        else
-          create_file manifest, SPROCKETS_SETUP
-        end
+        add_to_manifest(manifest('app/javascript'), IMPORTMAP_SETUP)
```

Comment:
Should be `BUN_SETUP` or we will use the same as importmap?

Reply:
I'll add a new constant